.expeditor/config.yml

```diff
@@ -79,21 +79,29 @@ merge_actions:
       only_if: built_in:bump_version
 
 subscriptions:
+  # the omnibus chain
   - workload: artifact_published:unstable:chef:{{version_constraint}}
     actions:
       - built_in:build_docker_image
   - workload: artifact_published:current:chef:{{version_constraint}}
     actions:
       - built_in:tag_docker_image
-      - built_in:promote_habitat_packages
   - workload: artifact_published:stable:chef:{{version_constraint}}
     actions:
       - built_in:rollover_changelog
       - bash:.expeditor/update_dockerfile.sh
       - built_in:tag_docker_image
-      - built_in:promote_habitat_packages
       - built_in:publish_rubygems
       - built_in:notify_chefio_slack_channels
+  # the habitat chain
+  - workload: hab_package_published:unstable:chef/chef-infra-client/*
```

Comment:
I would subscribe to https://expeditor.chef.io/docs/reference/workloads/#buildkite-hab-build-group instead of the package being published.